package.json

```diff
@@ -125,6 +126,7 @@
     "@saleor/app-sdk": "^0.34.1",
     "@types/apollo-upload-client": "^17.0.0",
     "@types/color-convert": "^2.0.0",
+    "@types/debug": "^4.1.7",
     "@types/fuzzaldrin": "^2.1.2",
     "@types/is-ci": "^3.0.0",
     "@types/jscodeshift": "^0.11.3",
```

Comment:
**suggestion**: what about adding `dev:debug` to npm scripts?

Reply:
I can if you like (I dont contribute do dashboard often so I will use only namespace I'm interested in anyway). WDYT @andrzejewsky ?

Reply:
@krzysztofzuraw we will discuss this among the team, let's not add this atm - during development you are interested only in very narrow things eg. the code you create, while you will get spam of the other logs, but we will discuss it